Simplenote/src/main/java/com/automattic/simplenote/authentication/SignInFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.automattic.simplenote.authentication
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.TextView
+import androidx.lifecycle.ViewModelProvider
+import com.automattic.simplenote.R
+import com.automattic.simplenote.utils.NetworkUtils
+import com.automattic.simplenote.viewmodels.MagicLinkRequestUiState
+import com.automattic.simplenote.viewmodels.RequestMagicLinkViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SignInFragment: MagicLinkableFragment() {
+
+    var viewModel: RequestMagicLinkViewModel? = null
```

Comment:
You can use the property delegate to build the view model:

```suggestion
    val viewModel: RequestMagicLinkViewModel by viewModels()
```